pkg/observ/stats.go

```diff
@@ -51,6 +62,33 @@ func registerPrometheusExporter(r *mux.Router, service string) error {
 	return nil
 }
 
+func registerStatsDataDogExporter(r *mux.Router, service string) (func(), error) {
```

Comment:
Do we need to pass `mux.Router` here? I think it was needed only for prometheus because it need to register Handler. In DataDog is not needed anymore because it not create any handler for that.

Reply:
That's right. I blatantly copied the prometheus exporter. Will fix.